### ISSUE TITLE
Can't add multiple rules() for multiple dynamic fields

### ISSUE THREAD

**Issue by dviantsoul** [jquery-validation/jquery-validation#2179]
Hello, I'm trying unsuccessfully to add two custom rules for two dynamically generated fields as such:

`<input type="text" name="volume1" id="volume1" class="volume_fields">`
`<input type="text" name="volume2" id="volume2" class="volume_fields">`

`<input type="text" name="lesson1" id="lesson1" class="lesson_fields">`
`<input type="text" name="lesson2" id="lesson2" class="lesson_fields">`

I need all the "class=volume_fields" to validate against some rules and "class=lesson_fields" to validate against other rules (I've simplified it to just "required" for this example). I can "add" one rule but when I try to add two I get an error. Either one works on its own but two do not. Also, interestingly, I doesn't throw an error if I don't use the "each" function, but it doesn't validate.

```
$('.volume_fields').each(function() {
    $(this).rules("add", {
        required: true,
        messages: {
            required: "Volume required",
         }
    });	
});
```
```
$('.lesson_fields').each(function() {
    $(this).rules("add", {
        required: true,
        messages: {
            required: "Lesson required",
         }
    });	
});
```

**Comment by dviantsoul** [comment]
Bueller? Bueller?

**Comment by gaosuccess** [comment]
Hi @dviantsoul , actually it works for me.

code sample is as follows just like yours:

```
<!doctype html>
<html>
    <head>
        <meta charset="utf-8" />
        <title>jquery validation demo of github #2179</title>
        <script src="jquery-3.3.1.js"></script>
        <script src="jquery.validate.js"></script>
        <script>
            $(function() {
                $('#myform').validate({});

                $('.must_required').each(function() {
                    $(this).rules('add', {
                        required: true,
                        messages: {
                            required: 'this is a must requried field'
                        }
                    });
                });

                $('.another_must_required').each(function() {
                    $(this).rules('add', {
                        required: true,
                        messages: {
                            required: 'this is another must requried field'
                        }
                    });
                });
            });
        </script>
    </head>

    <body>
        <form id="myform">
            <input name="field1" class="must_required" />
            <input name="field2" class="must_required" />
            <hr />
            <input name="field3" class="another_must_required" />
            <input name="field4" class="another_must_required" />
            <hr />
            <input type="submit" value="submit" />
        </form>
    </body>
</html>
```

**Comment by dviantsoul** [comment]
I tried this and it was giving me the same error. After much frustration I realized that if you provided this sample, it should work. This made me think that it was something else. Eventually I realized that I was working with an older version of Validate (v1.11). When I updated my copy of Validate both yours and my versions of the code worked. Thank you for your assistance.

**Comment by ace113** [comment]
the validation is working only on keyup. but when i try to submit the form it validates only first row and  ignores the rest of dynamically added field validations.
